browser/ui/views/commands/default_accelerators_mac.mm

```diff
@@ -53,6 +53,13 @@ AcceleratorMapping ToAcceleratorMapping(NSMenuItem* item) {
   NSString* keyEquivalent = item.keyEquivalent;
   DVLOG(2) << __FUNCTION__ << item.tag << " > "
            << base::SysNSStringToUTF16(keyEquivalent);
+
+  unsigned short keyCode = [keyEquivalent characterAtIndex:0];
+  if ([keyEquivalent isEqualToString:@"\t"]) {
```

Comment:
```suggestion
  if (keyCode == '\t') {
```
Curious that `\t` is the only case that we should do like above?

Reply:
Hmm, confusing.. first charactor could be something like `B` or `D`. and then `keyCode` var stores its ascii value.
In the `B` case, `keyCode` stores `66` and it's passed as `keyCode` params for creating `NSEvent`.
Does `keyCode` of `NSEvent` ctor accept ascii value?

Reply:
It seems `[keyEquivalent characterAtIndex:0]` is keyboard code. and it should be converted to keycode?

Reply:
Yeah, I think we should intialize keCode with -1.

Reply:
How about below? 
```
  const unsigned short charCode = [keyEquivalent characterAtIndex:0];
  const int keyCode =
      ui::MacKeyCodeForWindowsKeyCode(static_cast<ui::KeyboardCode>(charCode), /*flags=*/0,
                                      /*us_keyboard_shifted_character=*/nullptr,
                                      /*keyboard_character=*/nullptr);

  NSEvent* keyEvent = [NSEvent keyEventWithType:NSEventTypeKeyDown
                                       location:NSZeroPoint
                                  modifierFlags:0
                                      timestamp:0

...
```

Reply:
Yeah! that works great! Thanks!